Upstatement/craft-starter
# install script tweak

# Changes

Adds a conditional check in `copy_dotenv` function in the install script so that local `.env` files won't be overwritten if they exist when installing/reinstalling

## How To Test

- Run `./bin/install`
- Check that your `.env` file hasn't changed (and watch for the goofy message in terminal)
